windows/RNLocalize/RNLocalizeModule.cpp

```diff
@@ -0,0 +1,190 @@
+#include "pch.h"
+
+#include "RNLocalizeModule.h"
+
+#include "winrt/Windows.Globalization.h"
+
+using namespace winrt::Microsoft::ReactNative;
+
+namespace winrt::RNLocalize
+{
+
+std::vector<winrt::hstring> RNLocalizeModule::getLocaleNames()
+{
+    std::vector<winrt::hstring> locales;
+    for (winrt::hstring locale : winrt::Windows::Globalization::ApplicationLanguages::Languages())
```

Comment:
do these APIs need to be called from the UI thread? on newer RNW, modules run on the JS thread by default

Reply:
We've been doing it this way for over a year in our app without issue. And we have run into plenty of other UI thread contextual issues.  I don't see anything that indicates it needs or has to be.